modules/katello_lifecycle_environment.py

```diff
@@ -75,7 +75,7 @@
   katello_lifecycle_environment:
     username: "admin"
     password: "changeme"
-    server_url: "https://katello.example.com"
+    server_url: "https://foreman.example.com"
```

Comment:
Don't the names imply that Katello is required? But I'm fine with changing all katello modules to use katello.example.com too.